src/main/kotlin/test/ktortemplate/core/utils/ExposedQueryUtils.kt

```diff
@@ -0,0 +1,68 @@
+package test.ktortemplate.core.utils
+
+import org.jetbrains.exposed.sql.AutoIncColumnType
+import org.jetbrains.exposed.sql.BooleanColumnType
+import org.jetbrains.exposed.sql.Column
+import org.jetbrains.exposed.sql.Expression
+import org.jetbrains.exposed.sql.IntegerColumnType
+import org.jetbrains.exposed.sql.LongColumnType
+import org.jetbrains.exposed.sql.Op
+import org.jetbrains.exposed.sql.Query
+import org.jetbrains.exposed.sql.SortOrder
+import org.jetbrains.exposed.sql.Table
+import org.jetbrains.exposed.sql.TextColumnType
+import org.jetbrains.exposed.sql.VarCharColumnType
+import org.jetbrains.exposed.sql.compoundAnd
+import org.jetbrains.exposed.sql.compoundOr
+import org.jetbrains.exposed.sql.select
+import org.jetbrains.exposed.sql.selectAll
+import test.ktortemplate.core.model.FilterField
+import test.ktortemplate.core.model.SortField
+import test.ktortemplate.core.model.SortFieldOrder
+
+/**
+ * Exposed framework utilities to construct queries automatically from user inputs
+ */
+object ExposedQueryUtils {
```

Comment:
Might as well make these top level functions, since the `object` keyword here is not being used for its purpose.

Reply:
Yep 👍